docker/Dockerfile.prod

```diff
@@ -42,4 +42,4 @@ RUN pip install -r requirements.txt
 # Now install our code, which may change frequently
 COPY . /code/listenbrainz
 
-USER daemon
+# USER daemon
```

Comment:
I had to comment this out because with the user `daemon`, I wasn't able to create a log file in the container anywhere, it kept throwing `Permission denied` errors. I would love to learn of a better way to fix that.